plugins/apollo-explorer/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "@backstage/plugin-apollo-explorer",
+  "version": "0.0.0",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "license": "Apache-2.0",
+  "publishConfig": {
+    "access": "public",
+    "main": "dist/index.esm.js",
+    "types": "dist/index.d.ts"
+  },
+  "backstage": {
+    "role": "frontend-plugin"
+  },
+  "scripts": {
+    "start": "backstage-cli package start",
+    "build": "backstage-cli package build",
+    "lint": "backstage-cli package lint",
+    "test": "backstage-cli package test",
+    "clean": "backstage-cli package clean",
+    "prepack": "backstage-cli package prepack",
+    "postpack": "backstage-cli package postpack"
+  },
+  "dependencies": {
+    "@backstage/core-components": "^0.10.0-next.3",
+    "@backstage/core-plugin-api": "^1.0.4-next.0",
+    "@backstage/theme": "^0.2.16-next.1",
+    "@material-ui/core": "^4.9.13",
+    "@material-ui/icons": "^4.9.1",
+    "@material-ui/lab": "^4.0.0-alpha.57",
+    "react-use": "^17.2.4",
+    "@apollo/explorer": "^1.1.1",
+    "use-deep-compare-effect": "^1.8.1"
```

Comment:
Don't think we use this dependency do we? Can we remove it?

Reply:
Ya I only added this b/c I got an error that it was missing 🤷 let me quickly try running again w/out it

Reply:
<img width="1728" alt="Screen Shot 2022-07-18 at 10 04 18 AM" src="https://user-images.githubusercontent.com/5607577/179553914-273ad85c-7dcb-433c-99f4-8b09d2430bcd.png">

Reply:
Well that is annoying. OK. No worries, thanks for checking!